packages/create-content-sdk-app/src/templates/nextjs/.sitecore/import-map.ts

```diff
@@ -1,8 +1,9 @@
 // This file is auto-generated by the Sitecore Content SDK.
 // Below are built-in Content SDK imports neccessary for the import map
-import { combineImportEntries, defaultImportEntries } from '@sitecore-content-sdk/nextjs/tools';
+import { combineImportEntries, defaultImportEntries } from '@sitecore-content-sdk/nextjs/codegen';
+import { ImportEntry } from '@sitecore-content-sdk/core/codegen';
```

Comment:
It should be available in "/nextjs" package. We shouldn't access "core" directly